Makefile

```diff
@@ -0,0 +1,81 @@
+.PHONY: $(filter-out vendor bin/tools/psalm/vendor bin/tools/cs-fixer/vendor,$(MAKECMDGOALS))
+
+MYSQL_URL="mysql://root:root@mysql:3306/zenstruck_foundry?charset=utf8"
+POSTGRESQL_URL="postgresql://root:root@postgres:5432/zenstruck_foundry?charset=utf8"
+MONGO_URL="mongodb://mongo:mongo@mongo:27017/mongo?compressors=disabled&amp;gssapiServiceName=mongodb&authSource=mongo"
+SQLITE_URL="sqlite://./var/app.db"
+
+INTERACTIVE:=$(shell [ -t 0 ] && echo 1)
+ifdef INTERACTIVE
+	DC_EXEC=docker-compose exec
+else
+	DC_EXEC=docker-compose exec -T
+endif
+
+DOCKER_PHP=${DC_EXEC} php
+
+.DEFAULT_GOAL := help
+
+UID = $(shell id -u)
+
+DOCKER_IS_UP = $(shell docker-compose ps | grep "Up (healthy)")
+
+ARGS := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+$(eval $(RUN_ARGS):;@:)
+
+help:
+	@fgrep -h "###" $(MAKEFILE_LIST) | fgrep -v fgrep | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+tests: cs-fixer psalm phpunit-full ### Run full tests suite
+
+phpunit-full: vendor ### Run full PHPunit (MySQL + Mongo)
+	@$(eval filter ?= '.')
+	@${DC_EXEC} -e USE_FOUNDRY_BUNDLE=1 -e DATABASE_URL=${MYSQL_URL} -e MONGO_URL=${MONGO_URL} php vendor/bin/simple-phpunit --configuration phpunit-dama-doctrine.xml.dist --filter=$(filter)
+
+phpunit-fast: vendor ### Run PHPunit with SQLite
+	@$(eval filter ?= '.')
+	@${DC_EXEC} -e DATABASE_URL=${SQLITE_URL} php vendor/bin/simple-phpunit --configuration phpunit-dama-doctrine.xml.dist --filter=$(filter)
```

Comment:
I need to take a fresh look at https://github.com/zenstruck/foundry/pull/49. Would be nice to be able to run bundle tests with this too.

Reply:
I don't think it's the same issue. #49 fails even on the latest version.

Reply:
I'm closing this, since an issue exists.